hangb.rb

```diff
@@ -0,0 +1,262 @@
+  end
+
+# ------------------
+# Visual methods
+# m: "U, here's the method to draw_board, which I asked for earlier in the start method definition."
+# U: sweet. Hmmm, looks a little complicated. Explain.
+# m: " no prob, girl.
+#    "1st - instead of that traditional "hangman" figure which reminds me of the violent
+#            history against part of our human race because of ignorance and profit, let's 
+#            use the Star Trek Enterprise as it symbolizes well, you know the prime directive,etc..."
+#    "2nd -  let's start with the whole ship. As long as the user guesses right, the ship stays whole."
+#    "3rd -  when they guess wrong though, take a part of the ship away, and turn it the color
+#            listed after the dot."
+# U: gotcha.
+# m: "Also, about the three quotes. I'm using that to ask you to keep the ship drawing together, double
+# quotes around single quotes. Also, I put an additional backslash in front of backslashes in the
+# ASC II drawing. This "escapes" them, so the drawing displays correctly and doesn't skew."
+#
+  def draw_board
+    if @wrong_count == 0
+      puts """
+    ___________________          _-_
+    \\==============_=_/ ____.---'---`---.____
+                \\_ \\    \\----._________.----/
+                  \\ \\   /  /    `-_-'
+              __,--`.`-'..'-_
+             /____          ||
+                  `--.____,-'
+
+"""
+    elsif @wrong_count == 1
+      puts """
+                                  _-_
+                         ____.---'---`---.____
+                \\_ \\    \\----._________.----/
+                  \\ \\   /  /    `-_-'
+              __,--`.`-'..'-_
+             /____          ||
+                  `--.____,-'
+
+""".cyan
+    elsif @wrong_count == 2
+      puts """
+                                  _-_
+                         ____.---'---`---.____
+                    \\    \\----._________.----/
+                     \\   /  /    `-_-'
+              __,--`.`-'..'-_
+             /____          ||
+                  `--.____,-'
+
+      """.blue
+    elsif @wrong_count == 3
+      puts """
+                                  _-_
+                         ____.---'---`---.____
+                          \\----._________.----/
+                          /  /    `-_-'
+              __,--`.`-'..'-_
+             /____          ||
+                  `--.____,-'
+
+      """.yellow
+    elsif @wrong_count == 4
+      puts """
+                                  _-_
+                         ____.---'---`---.____
+                          \\----._________.----/
+                          /  /    `-_-'
+
+
+
+
+
+      """.magenta
+    elsif @wrong_count == 5
+      puts """
+                                  _-_
+                         ____.---'---`---.____
+                          \\----._________.----/
+                                  `-_-'
+
+
+
+
+
+      """.red
+    else
+      puts "Sorry,WARP CORE BREACH! You lose!".red.blink
+      exit
```

Comment:
This method is meant for drawing, so having game logic here (the `exit`) could be a little confusing.